modules/subscription/service_principal.tf

```
@@ -1,5 +1,5 @@
 resource "time_rotating" "one_year" {
```

possibly want to rename this resource too? and update it in where it's referenced?
```suggestion
resource "time_rotating" "rotation_period" {
```
or

```suggestion
resource "time_rotating" "this" {
```